packages/editor/src/components/document-tools/index.js

```diff
@@ -117,7 +113,6 @@ function DocumentTools( {
 				className
 			) }
 			aria-label={ toolbarAriaLabel }
-			shouldUseKeyboardFocusShortcut={ ! showFixedToolbar }
```

Comment:
Prevents rerender on editor document tools when block is selected. Shortcut works as intended without this prop.